stage #2/module #4 String/lesson #1. String/content.md

```diff
@@ -0,0 +1,200 @@
+| static String valueOf(char c)                                                              | Returns the string representation of the char argument.                                                                                     |
+| static String valueOf(char[] data)                                                         | Returns the string representation of the char array argument.                                                                               |
+| static String valueOf(char[] data, int offset, int count)                                  | Returns the string representation of a specific subarray of the char array argument.                                                        |
+| static String valueOf(double d)                                                            | Returns the string representation of the double argument.                                                                                   |
+| static String valueOf(float f)                                                             | Returns the string representation of the float argument.                                                                                    |
+| static String valueOf(int i)                                                               | Returns the string representation of the int argument.                                                                                      |
+| static String valueOf(long l)                                                              | Returns the string representation of the long argument.                                                                                     |
+| static String valueOf(Object obj)                                                          | Returns the string representation of the Object argument.                                                                                   |
+
+Constructors of **String**:
+
+| Constructor                                                      | Description                                                                                                                                                                                 |
+|------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| String()                                                         | Initializes a newly created String object so that it represents an empty character sequence.                                                                                                |
+| String(byte[] bytes)                                             | Constructs a new String by decoding the specified array of bytes using the platform's default charset.                                                                                      |
```

Comment:
Consider describing in a few words from what params you may create a string.
I believe, it is not a good idea to put 2 big tables in a row

Reply:
adjusted, thanks